server/src/main/kotlin/core/gameflow/Player.kt

```diff
@@ -0,0 +1,35 @@
+package core.gameflow
+
+import core.Card
+
+data class Player(
+    val position: Int,
+    val stack: Int,
+    val holeCards: List<Card>,
+    val chipsInPot: Int,
+    val currentBet: Int,
```

Comment:
Maybe we should add default values (empty list for holeCards and 0 for chipsInPot / currentBet).

Reply:
What about position, stack etc?

Reply:
Those should always be set up explicitly. The defaults that I mentioned apply when creating new Player instances at the begining of every new hand, that's why they may come in handy.